benchmark/test_unary_pointwise_perf.py

```diff
@@ -270,3 +288,316 @@ def test_bitwise_right_shift_perf():
         dtypes=INT_DTYPES,
     )
     bench.run()
+
+
+class SwigluBenchmarkResult(BenchmarkResult):
```

Comment:
Can we add a base class that can be used by TE ops benchmark test such as #1062, #1055, #1052, #1056